File: official/docs/java/current/billing/add-stripe-credit-card.java
package billing;

import com.easypost.exception.EasyPostException;
import com.easypost.model.PaymentMethod;
import com.easypost.service.EasyPostClient;

public class CreateStripeCreditCardSecret {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        PaymentMethodObject paymentMethod = client.referralCustomer.addCreditCardFromStripe(
                "REFERRAL_USER_API_KEY",
                "seti_...",
                PaymentMethod.Priority.PRIMARY
        );

        System.out.println(paymentMethod);
    }
}

File: official/docs/php/current/billing/add-stripe-credit-card.php
<?php

$client = new \EasyPost\EasyPostClient('EASYPOST_API_KEY');

$paymentMethod = $client->referralCustomer->addCreditCardFromStripe(
    'REFERRAL_USER_API_KEY',
    'seti_...',
    'primary',
);

echo $paymentMethod;
